src/ec2/commands.ts

```diff
@@ -30,3 +32,14 @@ export async function tryConnect(selection?: Ec2Selection): Promise<void> {
     const ec2Client = new Ec2ConnectionManager(selection.region)
     await ec2Client.attemptEc2Connection(selection)
 }
+
+export async function copyInstanceId(instanceId: string): Promise<void> {
+    try {
+        if (!instanceId) {
+            throw new ToolkitError(`Attempting to copy undefined instanceId.`)
```

Comment:
param is defined as string so shouldn't be undefined. The [global handler](https://github.com/aws/aws-toolkit-vscode/blob/926fca0d358bd36c473febf182e9c30e4a2af198/src/extension.ts#L279) is good enough here though, since this is unexpected, a special error is not worth the extra code.